google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/RateLimitingServerStreamingCallable.java

```diff
@@ -260,11 +274,11 @@ protected void onCompleteImpl() {
       outerObserver.onComplete();
     }
 
-    private void updateQps(double factor, Duration period) {
+    private void updateQps(double factor, Duration period, @Nullable Throwable t) {
```

Comment:
Maybe instead of throwable pass in the status. So you don't need to do the null check, you can just have OK status if the update is from a response

Reply:
But `updateQps` is used by both `onResponseImpl` and `onErrorImpl`